_scripts/templates/Go/tester.psm1

```diff
@@ -1,14 +1,20 @@
 # Template generated code from trgen <version>
 function Build-Grammar {
-<tool_grammar_files:{x |
-    $g = antlr <x> -Dlanguage=Go <antlr_tool_args:{y | <y> } >
+<tool_grammar_tuples:{x |
+    $g = antlr <x.GrammarFileName> -encoding <antlr_encoding> -Dlanguage=Go <x.AntlrArgs> <antlr_tool_args:{y | <y> } >
     if($LASTEXITCODE -ne 0){
         return @{
             Message = $g
             Success = $false
         \}
     \}
 }>
+    # Output version of pwsh.
+    #Get-Host | Select-Object Version | Write-Host
+    # Output environmental variables.
+    #dir env: | Out-String | Write-Host
+    # Output go version
+    #go version | Write-Host
```

Comment:
I'm adding all this "debugging" code in comments. Powershell is extremely difficult to use. Years of working with it have taught me that. Unless one programs in it every day, it's best to save the code to get basic debugging information out. I spent days trying out various codes to print debugging info. For example, in Bash, "export" just prints the list of environmental variables. It's been that way for ~30 years. What is the equivalent on Powershell? Well, if you look that up, you get dozens of answers in Stackoverflow. "dir env:" seems to be the most favored, but it never worked. The problem turned out to be the fact that Windows11 installs an ancient version of Powershell in c:/Windows/ that does not support "dir env:". YOU HAVE TO INSTALL THE LATEST PWSH.EXE!